### PR TITLE
Properly quote .Values.telemetry.v2.accessLogPolicy.logWindowDuration

### LINKED ISSUE #25550: logWindowDuration setting leads to Istio proxy errors
**Bug description**

Setting `.Values.telemetry.v2.accessLogPolicy.logWindowDuration` to a duration value produces incorrect JSON in the `stackdriver-sampling-accesslog-filter-1.6` EnvoyFilter, that leads to errors in Istio proxy.

The produced JSON is

```
...
value:
        name: istio.access_log
        typed_config:
          '@type': type.googleapis.com/udpa.type.v1.TypedStruct
          type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
          value:
            config:
              configuration: |
                {
                  "log_window_duration": 43200s
                }
              vm_config:
                code:
                  local:
                    inline_string: envoy.wasm.access_log_policy
...

```

`wasm log : Cannot parse AccessLog plugin configuration JSON string`

[ ] Configuration Infrastructure
[ ] Docs
[x] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### PR DESCRIPTION
**Pull request by cagataygurturk**
Fixes #25550

[ ] Configuration Infrastructure
[ ] Docs
[x] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by morvencao — APPROVED** [review]
/ok-to-test

Please run `make gen` to fix the test.

File: manifests/charts/istiod-remote/files/gen-istiod-remote.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ apiVersion: v1
 kind: ConfigMap
```
Comment (dgn) [inline manifests/charts/istiod-remote/files/gen-istiod-remote.yaml:4]: This file should not have changed

Comment (cagataygurturk) [reply manifests/charts/istiod-remote/files/gen-istiod-remote.yaml:4]: Yes, I have run make gen without latest changes from the upstream. Fixing now.

**Review by dgn — APPROVED** [review]

**Pull request merged**